### ISSUE TITLE
[Scheduler] Keyboard navigation is missing for the "show full day/ business hours" button

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#3268]
**Describe the bug**
Currently, there isn't a way to reach the "show full day/ business hours" button with the keyboard. 

**To Reproduce**
It can be tested in the Scheduler demos
https://www.telerik.com/kendo-angular-ui/components/scheduler/keyboard-navigation/

**Expected behavior**
To be able to navigate with the keyboard to the "show full day/ business hours" button

**Browser**
 - OS: [Windows 10]
 - Browser [Chrome]
 - Version [89.0.4389.114]
